src/AppBundle/Twig/AppExtension.php

```diff
@@ -28,9 +29,12 @@ class AppExtension extends \Twig_Extension
 {
     private $parser;
 
-    public function __construct(Markdown $parser)
+    private $purifier;
+
+    public function __construct(Markdown $parser, HTMLPurifierExtension $purifier)
```

Comment:
and I think this should be done in the Markdown class itself, by making ``AppBundle\Utils\Markdown`` a safe markdown parser